pytest_httpserver/httpserver.py

```diff
@@ -315,7 +355,8 @@ class HTTPServer:   # pylint: disable=too-many-instance-attributes
     DEFAULT_LISTEN_HOST = "localhost"
     DEFAULT_LISTEN_PORT = 4000
 
-    def __init__(self, host=DEFAULT_LISTEN_HOST, port=DEFAULT_LISTEN_PORT, ssl_context: Optional[SSLContext] = None):
+    def __init__(self, host=DEFAULT_LISTEN_HOST, port=DEFAULT_LISTEN_PORT, ssl_context: Optional[SSLContext] = None,
```

Comment:
Could you update the docstring with the added parameter?

Also, if you want you could also add a few lines and examples to the documentation about this feature. If not, I can do that.

Reply:
Please update the documentation. I'm not sure which examples you expect.